scheduler/scheduler_dfs.go

```diff
@@ -81,7 +81,7 @@ func (s *Scheduler) syncDfs(ctx context.Context, resolvedResources chan<- *schem
 	wg.Wait()
 }
 
-func (s *Scheduler) resolveTableDfs(ctx context.Context, table *schema.Table, client schema.ClientMeta, parent *schema.Resource, resolvedResources chan<- *schema.Resource, depth int) {
+func (s *Scheduler) resolveTableDfs(ctx context.Context, table *schema.Table, client schema.ClientMeta, parent *schema.Resource, resolvedResources chan<- *schema.Resource, depth int, syncOpts *SyncOptions) {
```

Comment:
These functions have a lot of parameters now but I think we can refactor this at a later point